src/include/distributed/placement_connection.h

```diff
@@ -23,7 +23,8 @@ extern MultiConnection * StartPlacementConnection(uint32 flags,
 												  const char *userName);
 
 extern void ResetPlacementConnectionManagement(void);
-extern void CheckForFailedPlacements(bool preCommit, bool using2PC);
+extern void MarkFailedShardPlacements(void);
+extern void PostCommitMarkFailedShardPlacements(bool using2PC);
```

Comment:
My first version did use a wrapper, however in that case the error messages emitted wer either super vague or incorrect for at least one call-site. This is the best factoring I could find which gives individualized error messages.